logs_monitoring.tf

```diff
@@ -1,13 +1,13 @@
 resource "aws_cloudformation_stack" "datadog-forwarder" {
-  name         = "${local.stack_prefix}datadog-forwarder"
+  name         = "${var.aws_region}-${local.stack_prefix}datadog-forwarder"
   capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM", "CAPABILITY_AUTO_EXPAND"]
   parameters = {
-    DdApiKeySecretArn = aws_secretsmanager_secret.datadog_api_key.arn
+    DdApiKeySecretArn = data.aws_secretsmanager_secret.dd_api_key.id
     DdApiKey          = "dummy-value"
-    DdTags            = "namespace:${var.namespace},env:${var.env}"
+    DdTags            = replace(jsonencode(var.aws_integration_tags), "/[{}\"]/", "")
     DdSite            = var.dd_forwarder_dd_site
     ExcludeAtMatch    = var.log_exclude_at_match
-    FunctionName      = "${local.stack_prefix}datadog-forwarder"
+    FunctionName      = "${var.aws_region}-${local.stack_prefix}datadog-forwarder"
```

Comment:
@ivancasco-sg  @mattbarlow-sg This avoids conflicts using the module in different regions in the same AWS account.

Reply:
Instead of having an `aws_region` variable, would it be possible to use the `aws_region` data source instead? This should pull the region from the provider. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/region